### DISCUSSION TITLE
Auto-pause during track recording

### DISCUSSION THREAD

**Discussion by mpavsic** [osmandapp/OsmAnd#16660] [Ideas]
Is there perhaps an auto-pause option for track recording? Some apps that I use for cycling have this nicely implemented and it works very well, however with OsmAnd I tend to get a "mess" of points when I stop for a longer time during trip recording and I don't manually pause. I know one can somehow tweak the settings to set when new points are added, but I have never managed to achieve such a nice result as with auto-pause setting in apps like Mapy.cz and Ride with GPS (to name just a few).

**Comment by pebogufi** [comment]
yes, you have just to change the setting:
Configure profile / trip recording / minimum displacement (10m)
may be also minimum accuracy (50m)
The values in () are an example I use.

**Reply by mpavsic** [reply]
Thanks for suggestions, I will retry. I had similar values there, but it didn't perform well (approx. 2 years ago), maybe something has changed since then.

**Reply by mpavsic** [reply]
Tried suggested settings, it worked rather well! Thanks!
